common/traits/wc_race_traits.txt

```diff
@@ -1447,6 +1449,8 @@ creature_qiraji = {
 		}
 	}
 	shown_in_ruler_designer = no
+	
+	life_expectancy = 130
```

Comment:
```suggestion
	life_expectancy = 930
```
They live for roughly a thousand years, yes?

Reply:
Resolved in bfb570b9d8ca2d317185b48da267329ccbb7fe84.